swagger/draft/provisioning_swagger.json

```diff
@@ -0,0 +1,619 @@
+          "404": {
+            "description": "not found",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "429": {
+            "description": "too many requests",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "500": {
+            "description": "internal server error",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "BulkResponse": {
+      "properties": {
+        "id": {
+          "type": "string"
+        },
+        "created": {
+          "type": "string"
+        },
+        "resourceType": {
+          "type": "string"
+        },
+        "resourceSubType": {
+          "type": "string"
+        },
+        "createdby": {
+          "type": "string"
+        },
+        "location": {
+          "type": "string"
+        }
+      }
+    },
+    "ProvisionResponse": {
+      "properties": {
+        "resourceSubType": {
+          "type": "string"
+        },
+        "createdBy": {
+          "type": "string"
+        },
+        "created": {
+          "type": "string"
+        },
+        "location": {
+          "type": "string"
+        },
+        "resourceType": {
+          "type": "string"
+        },
+        "originalRequest": {
+          "type": "object",
+          "items": {
+            "$ref": "#/definitions/BulkRequest"
+          }
+        }
+      }
+    },
+    "BulkOperation": {
+      "properties": {
+        "path": {
+          "type": "string"
+        },
+        "method": {
+          "type": "string"
+        },
+        "data": {
+          "type": "string"
+        }
+      }
+    },
+    "BulkRequest": {
+      "properties": {
+        "schemas": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "failOnErrors": {
+          "type": "integer",
+          "format": "int32"
+        },
+        "operations": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/BulkOperation"
+          }
+        }
+      }
+    },
+    "BulkRequestStatus": {
+      "properties": {
+        "completed": {
+          "type": "string"
+        },
+        "status": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/StatusResponse"
+          }
+        }
+      }
+    },
+    "StatusResponse": {
+      "properties": {
+        "operationId": {
+          "type": "string"
+        },
+        "extensions": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ExtensionResponse"
+          }
+        },
+        "completed": {
+          "type": "boolean"
+        }
+      }
+    },
+    "ExtensionResponse": {
+      "properties": {
+        "message": {
+          "type": "array"
+        },
```

Comment:
This object requires an items property because it's an array. See https://swagger.io/docs/specification/data-models/data-types/#array for reference.